swan-lake/why-ballerina/graphical.md

```diff
@@ -1,67 +1,29 @@
 title: Graphical
-description: See why the support for a graphical model lays the foundation for designing the syntax and semantics of the Ballerina programming language.
+description: See why the support for a visual representation of the development experience lays the foundation for designing the syntax and semantics of the Ballerina programming language.
 keywords: ballerina, programming language, sequence diagram, graphical, diagram editor, why ballerina
+intro: See why the support for a visual representation of the development experience lays the foundation for designing the syntax and semantics of the Ballerina programming language.
 permalink: /why-ballerina/graphical/
 active: graphical
 ---
 
-In today’s cloud-era, you need technologies that can model distributed systems in a more developer-friendly way. This means that for a single use case you need to model a flow that shows how multiple actors interact with each other, how concurrent execution flows, and what remote endpoints are involved. Sequence diagrams are known to be the best way to visually describe this.
+In today’s cloud-era, you need technologies that can model distributed systems in a more developer-friendly way. The feature-rich development experience of the [Ballerina Visual Studio code extension](/vscode/) brings in graphical representations for designing the architecture of your applications, designing and editing your services, modeling how concurrent execution flows and how multiple actors interact with each other, and mapping the data conversions within your applications.
 
-That’s why sequence diagrams are the foundation for designing the syntax and semantics of the Ballerina language. Ballerina provides the flexibility of a general-purpose language while having features to model and visualize solutions based on higher-level abstractions derived from sequence diagrams.
+Ballerina provides the flexibility of a general-purpose language with the capabilities to model and visualize solutions based on higher-level abstractions derived from the graphical representations provided by its developer tooling features as described below.
 
+## Architecture View
 
-## Sequence diagrams in Ballerina
+The [Architecture View](https://wso2.com/ballerina/vscode/docs/design-the-application/) of the Ballerina VS Code extension provides a holistic picture of the components and their interactions in Ballerina projects purely based on the source code. It provides several types of diagrams, namely, the Service Diagram Level 1 (Services), Service Diagram Level 2 (Resources), type diagram, and composition diagram to explore and ease the designing of the application.  
 
-Being based on sequence diagrams, Ballerina allows you to visualize a program written in Ballerina as a sequence diagram. The diagram will display the logic and network interaction of a function or a service resource. You can view and edit these diagrams using the Ballerina VSCode plugin.
+## Service designing
```

Comment:
Why this is a action while other topics are VSCode component?

Reply:
Updated.